src/components/PopularityFilterCollapse.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Collapse, Checkbox } from "@baltimorecounty/dotgov-components";
+
+const PopularityFilterCollapse = props => {
+  const { id, header, onChange, isChecked, isExpanded } = props;
+
+  return (
+    <Collapse id={id} header={header}>
+      <Checkbox
+        id={`${id}Checkbox`}
+        name={`${id}Checkbox`}
+        onChange={onChange}
+        checked={isChecked}
+        isExpanded={isExpanded}
```

Comment:
the isExpaned attribute is on the checkbox but is meant for the collapse.

Reply:
removed since it defaults to expanded